client/packages/system/src/Stock/Components/StockLineForm.tsx

```diff
@@ -149,309 +153,334 @@ export const StockLineForm = ({
+                  Input={
+                    <NumericTextInput
+                      width={160}
+                      value={draft.volumePerPack ?? 0}
+                      decimalLimit={10}
+                      onChange={volumePerPack => onUpdate({ volumePerPack })}
+                    />
+                  }
                 />
-              }
-            />
-            {!packEditable && (
-              <StyledInputRow
-                label={t('label.total-volume')}
-                Input={
-                  <NumericTextInput
-                    disabled
-                    width={160}
-                    decimalLimit={10}
-                    // Need to coalesce to 0 to avoid NaN while user is editing volumePerPack!
-                    value={
-                      (draft.volumePerPack ?? 0) * draft.totalNumberOfPacks
+                {!packEditable && (
+                  <StyledInputRow
+                    label={t('label.total-volume')}
+                    Input={
+                      <NumericTextInput
+                        disabled
+                        width={160}
+                        decimalLimit={10}
+                        // Need to coalesce to 0 to avoid NaN while user is editing volumePerPack!
+                        value={
+                          (draft.volumePerPack ?? 0) * draft.totalNumberOfPacks
+                        }
+                      />
                     }
                   />
-                }
-              />
-            )}
-            <TextWithLabelRow
-              label={t('label.supplier')}
-              text={String(supplierName)}
-              textProps={{ textAlign: 'end' }}
-            />
-            {showVVMStatus && (
-              <StyledInputRow
-                label={t('label.vvm-status')}
-                labelWidth={isNewModal ? '212px' : null}
-                Input={
-                  <VVMStatusSearchInput
-                    selected={draft?.vvmStatus ?? null}
-                    onChange={vvmStatus => onUpdate({ vvmStatus })}
-                    disabled={!isNewModal}
-                    width={!isNewModal ? 160 : undefined}
-                    useDefault={isNewModal}
+                )}
+                <TextWithLabelRow
+                  label={t('label.supplier')}
+                  text={String(supplierName)}
+                  textProps={{ textAlign: 'end' }}
+                />
+                {showVVMStatus && (
+                  <StyledInputRow
+                    label={t('label.vvm-status')}
+                    labelWidth={isNewModal ? '212px' : null}
+                    Input={
+                      <VVMStatusSearchInput
+                        selected={draft?.vvmStatus ?? null}
+                        onChange={vvmStatus => onUpdate({ vvmStatus })}
+                        disabled={!isNewModal}
+                        width={!isNewModal ? 160 : undefined}
+                        useDefault
```

Comment:
needs to keep the `useDefault={isNewModal}` here - should only use default values when initially creating the stock line